check/all_windows.go

```diff
@@ -6,4 +6,16 @@
 
 package check
 
+import "runtime"
+
 const pathSeperator = ";"
+
+const windowsOS = "windows"
+
+func isWindows() bool {
+	if runtime.GOOS == windowsOS {
```

Comment:
`return runtime.GOOS == windowsOS`

Reply:
You're right, i don't know what i was thinking.
:D